### PR TITLE
store/tikv: remove tidb/errno dependency

### PR DESCRIPTION
**Pull request by raygift**
### What problem does this PR solve?

remove store/tikv 's dependency on tidb/errno 
Part of #22513

Problem Summary:

### What is changed and how it works?

What's Changed:

remove tidb/errno dependency from store/tikv
add store/tikv/errno

### Check List 

Tests 

- Unit test

### Release note 

- No release note

### CODE REVIEW

**Comment by raygift** [comment]
@disksing  PTAL

**Comment by disksing** [comment]
There are many errors are not required by tikv package.

**Comment by raygift** [comment]
> There are many errors are not required by tikv package.

remove unused error consts in store/tikv/errno in latest commit ,PTAL

**Review by disksing — APPROVED** [review]

File: store/tikv/errno/errcode.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 PingCAP, Inc.
```
Comment (AndreMouche) [inline store/tikv/errno/errcode.go:1]: 2021？

File: store/tikv/kv/error.go [hunk 2]
```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/pingcap/errors"
 	"github.com/pingcap/kvproto/pkg/kvrpcpb"
 	"github.com/pingcap/kvproto/pkg/pdpb"
-	mysql "github.com/pingcap/tidb/errno"
+	mysql "github.com/pingcap/tidb/store/tikv/errno"
```
Comment (AndreMouche) [inline store/tikv/kv/error.go:20]: Maybe we can consider moving this file into package `tikv/errno` in the future?

**Review by AndreMouche — COMMENTED** [review]
Rest LGTM

File: store/tikv/errno/errcode.go (hunk 1 above)
Comment (raygift) [reply store/tikv/errno/errcode.go]: got it, modified the year in last commit

**Review by AndreMouche — APPROVED** [review]
LGTM

**Pull request merged**